frontend/components/Slider/index.jsx

```diff
@@ -119,6 +124,14 @@ const mapStateToProps = (state, props) => {
     productId: props.productId,
     type: props.type,
   };
+
+  if (params.type === 'property') {
```

Comment:
```suggestion
  if (params.type === 'TYPE_PROPERTY') {
```
do the same for all other places